corehq/apps/domain/_design/views/by_alias/map.js

```diff
@@ -0,0 +1,5 @@
+function(doc) {
```

Comment:
I thought moving them to an app just made it quicker to sync when developing. If it makes production indexing/whatever faster, happy to move them.